packages/manager/modules/pci/src/projects/project/storages/databases/database/users/informations/informations.controller.js

```diff
@@ -0,0 +1,23 @@
+import { map } from 'lodash';
+
+export default class usrInfomrationCtrl {
+  /* @ngInject */
+  constructor($translate, DatabaseService) {
+    this.$translate = $translate;
+    this.service = DatabaseService;
+  }
+
+  $onInit() {
+    this.isLoading = false;
+    this.mapArrayForChip('keys');
+    this.mapArrayForChip('categories');
+    this.mapArrayForChip('channels');
+    this.mapArrayForChip('commands');
```

Comment:
Move 'keys', 'categories' etc to a constant file. If it is an array, do a `array.forEach((ele) => this.mapArrayForChip(ele));`

Reply:
Done here: https://github.com/ovh/manager/pull/5488/commits/45d16da89ff2c70503a9a111c3fd5996a5eecbe1